src/node_v8.cc

```diff
@@ -20,25 +20,37 @@ using v8::Uint32;
 using v8::V8;
 using v8::Value;
 
+#define HEAP_STATISTICS_PROPERTIES(V)                                         \
+  V(0, total_heap_size, kTotalHeapSizeIndex);                                 \
+  V(1, total_heap_size_executable, kTotalHeapSizeExecutableIndex);            \
+  V(2, total_physical_size, kTotalPhysicalSizeIndex);                         \
+  V(3, used_heap_size, kUsedHeapSizeIndex);                                   \
+  V(4, heap_size_limit, kHeapSizeLimitIndex);
+
+#define kHeapStatisticsBufferLength 5
+#define kHeapStatisticsBufferType v8::kExternalUint32Array
```

Comment:
This is not exactly wrong but try to prefer `static const size_t x = 42` over `#define x 42`.  You can even turn it into something fancy like:

```c++
#define V(a, b, c) +1
static const size_t kHeapStatisticsBufferLength = HEAP_STATISTICS_PROPERTIES(V);
#undef V
```

It looks horrible but it's pretty bulletproof against future changes.